lcli/src/main.rs

```diff
@@ -695,13 +730,42 @@ fn main() {
                         .long("count")
                         .value_name("COUNT")
                         .takes_value(true)
+                        .required(true)
+                        .help("Produces validators in the range of 0..count."),
+                )
+                .arg(
+                    Arg::with_name("base-dir")
+                        .long("base-dir")
+                        .value_name("BASE_DIR")
+                        .takes_value(true)
+                        .required(true)
+                        .help("The base directory where validator keypairs and secrets are stored"),
+                )
+                .arg(
+                    Arg::with_name("node-count")
+                        .long("node-count")
+                        .value_name("NODE_COUNT")
+                        .takes_value(true)
+                        .help("The number of nodes to divide the validator keys to"),
+                )
+        )
+        .subcommand(
+            SubCommand::with_name("mnemonic-validators")
+                .about("Produces validator directories by deriving the keys from a mnemonic.")
```

Comment:
```suggestion
                .about("Produces validator directories by deriving the keys from \
                        a mnemonic. For testing purposes only, DO NOT USE IN \
                        PRODUCTION!")
```

I think what you've done *is* actually production-safe, however I figure we might end up getting a bit fast and loose with `lcli` stuff in the future ☺️